pages/ksphere/kommander/latest/clusters/index.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: layout.pug
+navigationTitle: Clusters
```

Comment:
@russurquhart1  can we rename this file to `Manage Clusters` or `Managing Clusters`? The word Clusters doesn't imply much in the docs navbar but the page has instructions full of lifecycle management